Sprint-2/implement/lookup.js

```diff
@@ -1,5 +1,14 @@
-function createLookup() {
-  // implementation here
+function createLookup(countryCurrencyPairs) {
+  const lookup = {};
+
+  for (let i = 0; i < countryCurrencyPairs.length; i++) {
+    const countryCode = countryCurrencyPairs[i][0];
```

Comment:
This works, but probably isn't the most clear way of iterating across this array.

Can you think how other looping constructs, or destructuring, could help you make this more clear and easy to read?